Makefile

```diff
@@ -73,4 +73,7 @@ clean:
 clean-full:
 	stack clean --full
 
+stack-yaml:
+	ros -Q scripts/yaml-generator.lisp
```

Comment:
This ensures it works out of the box, right?

Reply:
yes, as we all have `ros` on our machines due to the org generator. This just tells it to run it when you say `make stack-yaml`